flask_project/database_utils.py

```diff
@@ -0,0 +1,186 @@
+"""
+database_utils.py
+
+This module stands for database management functions.
+Adding new records, retrieving all records etc.
+"""
+from datetime import datetime, timedelta
+from flask_sqlalchemy import SQLAlchemy
+from .models import db, URL, IP_address, Domains
+
+
+
+def create_ip_record(ipqs_data: dict):
+    """
+    Create an IP address record in the database.
+    Args:
+        - ipqs_data (dict): A dictionary containing data about ip address,
+        from IP Quality Score Service.
+    """
+    ip_address_obj = IP_address(
+        ip_address=ipqs_data["ip_address"],
+        record_created_at=datetime.now(),
```

Comment:
I think better to use one datetime object and calculate now() only once in this case.